### PR TITLE
HW 20 gamzinova

### PR DESCRIPTION
**Pull request by gamzinova**
Hi, Please find attached my homework 20.
Thanks in advance for reviewing!

Best,
Anna

### CODE REVIEW

File: Anna_Gamzinova/api_collections/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pytest==7.2.0
+requests==2.28.1
+selenium==4.6.0
+
```
Comment (bohdandovbsyh) [inline Anna_Gamzinova/api_collections/requirements.txt:4]: looks like duplication

File: Anna_Gamzinova/api_collections/test_api.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    assert response.status_code == HTTPStatus.OK, f'Status code is not as expected\n Actual: {response.status_code}' \
+                                                  f'\nExpected: {HTTPStatus.OK}'
+
+
+def test_create_person():
+    new_user = {
+        "name": "Santa Clause",
+        "gender": "male",
+        "email": "hohoho1@gmail.com",
+        "status": "active"}
```
Comment (bohdandovbsyh) [inline Anna_Gamzinova/api_collections/test_api.py:24]: it is better to replace it with a data class.

File: Anna_Gamzinova/api_collections/test_api.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+def test_create_person_name_check():
+    """
+    A test checks that the user ws created with the requested name
+    """
+    new_user = {
+        "name": "Mickie Mouse",
+        "gender": "male",
+        "email": "hellomickie1@gmail.com",
+        "status": "active"}
```
Comment (bohdandovbsyh) [inline Anna_Gamzinova/api_collections/test_api.py:39]: same here

File: Anna_Gamzinova/api_collections/test_api.py (hunk 2 above)
Comment (gamzinova) [reply Anna_Gamzinova/api_collections/test_api.py]: fixed

File: Anna_Gamzinova/api_collections/test_api.py (hunk 3 above)
Comment (gamzinova) [reply Anna_Gamzinova/api_collections/test_api.py]: fixed

**Pull request merged**